src/bentoml/client.py

```diff
@@ -27,15 +31,19 @@ def __init__(self, svc: Service, server_url: str):
 
         for name, api in self._svc.apis.items():
             if not hasattr(self, name):
-                setattr(self, name, functools.partial(self._call, _bentoml_api=api))
+                setattr(self, name, functools.partial(self._sync_call, _bentoml_api=api))
+                setattr(self, f"async_{name}", functools.partial(self._call, _bentoml_api=api))
 
-    def call(self, api_name: str, inp: t.Any = None, **kwargs: t.Any) -> t.Any:
-        asyncio.run(self.async_call(api_name, inp))
+    def call(self, bentoml_api_name: str, inp: t.Any = None, **kwargs: t.Any) -> t.Any:
+        return asyncio.run(self.async_call(bentoml_api_name, inp))
 
     async def async_call(
-        self, api_name: str, inp: t.Any = None, **kwargs: t.Any
+        self, bentoml_api_name: str, inp: t.Any = None, **kwargs: t.Any
     ) -> t.Any:
-        return self._call(inp, _bentoml_api=self._svc.apis[api_name])
+        return await self._call(inp, _bentoml_api=self._svc.apis[bentoml_api_name])
+
+    def _sync_call(self, inp: t.Any = None, *, _bentoml_api: InferenceAPI, **kwagrs: t.Any):
```

Comment:
where is this function getting called?

Reply:
It gets used for the `.<API name>` methods.

Reply:
ah got it. I didn't see the `setattr`.